bin/GameIndex.dbf

```diff
@@ -27489,6 +27491,7 @@ Region = NTSC-J
 Serial = SLPM-66694
 Name   = Spongebob
```

Comment:
You forgot to change the name here as well.

Reply:
No, I just reverted that because adding the full title would be incorrect for this region.

Reply:
It might get confusing with other Spongebob titles this way. I don't know how many there are.

Reply:
It's the only Japanese Spongebob title. Japanese people will recognise the game with the current name (the Japanese cartoon title also lacks Squarepants).